libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationViewModel.java

```diff
@@ -212,9 +215,19 @@ private void initUnitType(NavigationUiOptions options) {
     unitType = options.directionsRoute().routeOptions().voiceUnits();
   }
 
+  private void initTimeFormat(MapboxNavigationOptions options) {
+    timeFormatType = options.timeFormatType();
+  }
+
   private void initVoiceInstructions(NavigationViewOptions options) {
-    boolean languageSupported = options.directionsRoute().voiceLanguage() != null;
-    instructionPlayer = new NavigationInstructionPlayer(getApplication(), language, languageSupported, accessToken);
+    boolean voiceLanguageSupported = options.directionsRoute().voiceLanguage() != null;
```

Comment:
Minor thing - Per consistency with other `boolean`s in the code base, what about calling it `isVoiceLanguageSupported` instead?